src/components/atoms/Pre/Pre.styled.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 export const StyledPre = styled.pre`
-  padding: 12px 40px 12px 12px;
+  padding: 12px 0 12px 12px;
```

Comment:
Wouldn't be better to keep the `padding` both on `left` and `right` side?

```suggestion
  padding: 12px;
```

Reply:
On the screenshot it looks like it's not redundant - there is no space between a button and end of text

Reply:
We already have padding on the button to the right. IMO it's totally sufficient to have padding only there. Ofc it's not a big deal to add this padding there.

Reply:
It's padding inside the button, not the margin outside. I'm not a designer though, so probably better to ask @fivenp :)